packages/datepickers/src/styled/StyledDay.ts

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { DefaultTheme, ThemeProps } from 'styled-components';
+import { retrieveComponentStyles, getColor, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+interface IStyledDayProps extends ThemeProps<DefaultTheme> {
+  isPreviousMonth?: boolean;
+  isToday?: boolean;
+  isDisabled?: boolean;
+  isSelected?: boolean;
+  isSmall: boolean;
```

Comment:
I noticed @jzempel commented on some sizing props elsewhere in this PR about using `isCompact` rather than `isSmall`. Would that apply to this line of code as well?